webrtc.html

```diff
@@ -5065,6 +5065,28 @@ <h2>RTP Media API</h2>
       {{MediaStreamTrack}} and {{RTCRtpReceiver}} are
       surfaced to the application via the {{track}} event.
     </p>
+    <p>
+      In order for an {{RTCRtpTransceiver}} to send and/or receive media with
+      another endpoint this must be negotiated with SDP such that both endpoints
+      have an {{RTCRtpTransceiver}} that is [= associated =] with the same
+      [= media description =]. When creating an offer, the local side will
+      associate all unassociated transceivers with a media description in the
+      offer. When the remote side sets the offer, media descriptions that are
+      not associated with a transceiver yet will get associated with a
+      transceiver. In order for the remote side to associate a transceiver with
+      a media section in this step, it has to create a <i>new</i> transceiver,
```

Comment:
I think this explains the remedy before the problem is fully understood. I'd focus on describing what happens and its limitations first, and move explanations of remedies to the very tend